### DISCUSSION TITLE
Modal won't open on android Brave Browser

### DISCUSSION THREAD

**Discussion by Alim32** [WalletConnect/walletconnect-monorepo#2874] [Developer Support]
Hey

I was testing around with the latest version, and noticed that on my samsung galaxy s21 with brave browser the modal often just won't open.

It works fine with Brave desktop and any other mobile (dapp) browser.

**Comment by glitch-txs** [comment]
I was able to reproduce Brave browser on Android. Seems like the provider doesn't initialize

**Comment by 0xAlexander** [comment]
> I was able to reproduce Brave browser on Android. Seems like the provider doesn't initialize

Any update on this issue? Users still can't connect to our dapp via web3modal when using Brave Browser on Android devices.
